components/org.wso2.carbon.identity.recovery/pom.xml

```diff
@@ -132,17 +132,6 @@
             <groupId>org.wso2.carbon.identity.governance</groupId>
             <artifactId>org.wso2.carbon.identity.multi.attribute.login.service</artifactId>
         </dependency>
-        <dependency>
```

Comment:
This dependency was added through https://github.com/wso2-extensions/identity-governance/pull/835. 
Removing since causing a circular dependency.